nCr mod P/nCr_mod_P.kt

```diff
@@ -0,0 +1,47 @@
+import java.util.*
+
+// Function for nCr
+fun nCr(n: Int, r: Int, mod: Int):Int {
+    if(n < r) {
+        return -1
+    }
+    var pascal = IntArray(r + 1)
+    pascal[0] = 1
+    for (i in 1 until r+1) {
+        pascal[i] = 0
+    }
+    // We use the known formula nCr = (n-1)C(r) + (n-1)C(r-1) for computing the values.
```

Comment:
A blank line before comment would be better.

Reply:
Done